ui/command_tab_view.cpp

```diff
@@ -45,21 +45,32 @@ CommandTabView::CommandTabView(const Dive::CommandHierarchy &command_hierarchy,
     m_command_buffer_view->header()->moveSection(2, 1);
 
     m_search_trigger_button = new QPushButton;
+    m_search_trigger_button->setObjectName("command_buffer_search_button");
     m_search_trigger_button->setIcon(QIcon(":/images/search.png"));
 
     QHBoxLayout *options_layout = new QHBoxLayout();
     options_layout->addWidget(m_search_trigger_button);
     options_layout->addStretch();
 
+    m_search_bar = new SearchBar(this);
+    m_search_bar->setObjectName("command_buffer_search_bar");
```

Comment:
can we define a const string for this name since it is used multiple times in this file? (just in case someone need to change this in the future, he/she doesn't have to search and modify the name in multiple places)

Reply:
Added object_names.h to hold these object names and any future ones. Currently these two names are used in command_tab_view and in main_window.